src/main/resources/archetype-resources/pom.xml

```diff
@@ -47,7 +47,8 @@
                 <artifactId>maven-antrun-plugin</artifactId>
                 <configuration>
                     <tasks>
-                        <copy file="target/${artifactId}-${version}.jar" todir="${spigot}/plugins"/>
+                        <property name="spigot.basedir" location="${spigot}"/>

```

Comment:
The new version is not OS specific anymore.
The location attribute for the property element convert \ and /. There are more explanation on ant property documentation page http://ant.apache.org/manual/Tasks/property.html